frontend/src/Css.tsx

```diff
@@ -315,6 +315,10 @@ export const commonCss = stylesheet({
   },
 });
 
+export const tailwindcss = {
+  sideNavItem: 'flex flex-row flex-shrink-0',
```

Comment:
Do we need a common style file for non reusable styles? It seems to me that the tailwind css way is to directly write these classes in the component.

Reply:
It makes sense to move it to the same file where it uses.

Yes I struggled whether I should extract the tailwind class names or just define them in place. I eventually tend to create a const for being consistent with the rest of classNames approaches in the same file.